db/migrate/20210917135713_create_sample_stocks.rb

```diff
@@ -1,10 +1,3 @@
 class CreateSampleStocks < ActiveRecord::Migration[6.0]
-  def change
-    create_table :sample_stocks do |t|
-      t.string :market_symbol
-      t.float :curr_price
-      t.string :logo_url
-      t.timestamps
-    end
-  end
+  #removed sample stock table transfered to market table
```

Comment:
Do you have this table on your database @michael-dollosa ? If yes, then @mumble-07 , you should create a new migration for dropping the table instead.

Reply:
was thinking since I'll be using a new brach to just setup db again. but this is good practice @mumble-07 just create migration for dropping